packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/palette/DiagramPalette.tsx

```diff
@@ -36,7 +36,7 @@ export const DiagramPalette = ({ targetObjectId }: DiagramPaletteProps) => {
 
   return isOpened ? (
     <DiagramPalettePortal>
-      <div className={classes.toolbar} style={{ position: 'absolute', left: x, top: y }}>
+      <div className={classes.toolbar} style={{ position: 'absolute', left: x ?? undefined, top: y ?? undefined }}>
```

Comment:
Is undefined really an expected value? Why would x/y be undefined? Do we want to open the palette in this situation.

Reply:
Done